### PR TITLE
Updated the t5_hf_summarization.py

### PR DESCRIPTION
**Pull request by seungjun-green**
made 3 commits:

**commit-1**

Changed 't5-base' to 't5-small'
In 'Define certain variables' section the MODEL_CHECKPOINT is defined as "t5-small" and also based on this console output 'Downloading:   0%|          | 0.00/231M [00:00<?, ?B/s]',  it seems "t5-small" is the right.

**commit-2**

Removed 'validation' text
'''
raw_datasets = raw_datasets.train_test_split(
    train_size=TRAIN_TEST_SPLIT, test_size=TRAIN_TEST_SPLIT
)
'''

Here we divided out raw_datasets in to training and test set with no validation set. So I think removing the word 'validation' here is right.

**commit-3**

fixed spelling error
Changed 'me summarized' to 'be summarized'

### CODE REVIEW

**Review by fchollet — COMMENTED** [review]
Thanks for the PR! Please replicate the fixes in the md and ipynb files.

**Comment by seungjun-green** [comment]
Done :)

**Review by fchollet — APPROVED** [review]
LGTM, thanks!

**Review by fchollet — APPROVED** [review]
LGTM, thanks!
